python/tk-core/README.md

```diff
@@ -1,4 +1,8 @@
-[![Build Status](https://secure.travis-ci.org/shotgunsoftware/tk-core.png?branch=master)](http://travis-ci.org/shotgunsoftware/tk-core)
```

Comment:
How can I do that ?  Running the tk_core_update.sh script also does some git stuff...

If I want Francis to use this branch for his QA, it tk-core code needs to be there.